pkg/db/db.go

```diff
@@ -444,7 +444,14 @@ func (h *DBHandler) DBSelectAnyRelease(ctx context.Context, tx *sql.Tx) (*DBRele
 		ctx,
 		selectQuery,
 	)
-	return h.processReleaseRow(ctx, err, rows)
+	processedRows, err := h.processReleaseRows(ctx, err, rows)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Why are we using the function to process multiple rows? Your query has a 'LIMIT 1 ', so the number of rows should either be 0 (processedRows = nil) or 1

Reply:
Further in the code, I've introduced a new function `DBSelectReleasesByApp` , which should return a list of `DBReleaseWithMetaData` So I needed a function to process multiple rows.
Still we can have a function to process one row, but that would create a lot of repetitive code.